backend/devices/bitbox02/keystore.go

```diff
@@ -51,15 +51,14 @@ func (keystore *keystore) RootFingerprint() ([]byte, error) {
 	return keystore.device.RootFingerprint()
 }
 
-// SupportsAccount implements keystore.Keystore.
-func (keystore *keystore) SupportsAccount(coin coin.Coin, meta interface{}) bool {
+// SupportsCoin implements keystore.Keystore.
+func (keystore *keystore) SupportsCoin(coin coin.Coin) bool {
 	switch specificCoin := coin.(type) {
 	case *btc.Coin:
 		if (coin.Code() == coinpkg.CodeLTC || coin.Code() == coinpkg.CodeTLTC) && !keystore.device.SupportsLTC() {
```

Comment:
This is confusing. I see `coin.Xxx` and `coinpkg.Xxx` but it's the same package, imported twice.

Reply:
oops, fixed